bots/peanutconversionbot/peanutconversion.bot.js

```diff
@@ -0,0 +1,18 @@
+const Slack = require('pico-slack')
+
+const peanut = require('./peanutfunctions.js')
+
+const messageHandler = (msg) => {
+	const weights = peanut.findWeights(msg.text)
+	if(weights.length < 1) return
+	const convertedWeights = peanut.convertWeights(weights)
+	const outgoingMsg = peanut.constructMessage(convertedWeights)
+	Slack.send(msg.channel, outgoingMsg)
+}
+
+module.exports = {
+	messageHandler,
+	load: () => {
```

Comment:
Why do you need this? The onMessage event listener can just be loaded on bot require right?

Reply:
I added it to prevent the listener from being created and leaking during test runs. But you pointed out that I can just stub it out, so I'll do that.

Reply:
I was already stubbing the `onMessage()` function, so there was nothing to do!